test/views/html/employments/AddEmploymentErrorPageViewSpec.scala

```diff
@@ -23,12 +23,12 @@ import controllers.routes
 class AddEmploymentErrorPageViewSpec extends TaiViewSpec {
   "Display error page when employment cannot be added" must {
     behave like pageWithTitle(messages("tai.addEmployment.employmentErrorPage.title"))
-    behave like pageWithCombinedHeader(
+    behave like pageWithCombinedHeaderNewTemplate(
```

Comment:
this reminds me we should also change the methods not to have newtemplate on them later :)